.changeset/wet-paws-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/config': patch
+---
+
+Ensure Config type reflects new get<T> & getOptional<T> function calls from [a1f5e65](https://github.com/backstage/backstage/pull/4620/commits/a1f5e65452faafe1e8772f8c81ea9b26bf83575f).
```

Comment:
Would say it'd be better to just modify the existing changeset, or even just piggyback on it an leave it at is. This fix doesn't really need a separate changelog entry since it's a fix for something that was introduced in the same release

Reply:
If that's valid sure :)

Reply:
Done